pkg/sidecar/appconf.go

```diff
@@ -179,22 +179,27 @@ func initFileQuery(cfg *Config, gtidPurged string) []byte {
 		[]string{"CREATE", "SELECT", "DELETE", "UPDATE", "INSERT"}, fmt.Sprintf("%s.%s", toolsDbName, toolsHeartbeatTableName),
 		[]string{"REPLICATION CLIENT"}, "*.*")...)
 
-	if len(gtidPurged) != 0 {
-		// If the xtrabackup information has GTID_PURGED then insert it into a table
-		// nolint: gosec
-		queries = append(queries, fmt.Sprintf(`
+	// create the status table used by the operator to configure or to mask MySQL node ready
+	// nolint: gosec
+	queries = append(queries, fmt.Sprintf(`
 		CREATE TABLE IF NOT EXISTS %[1]s.%[2]s (
-            id int PRIMARY KEY,
-            gtid varchar(512) NOT NULL,
-            used BOOLEAN DEFAULT false
-		)`, constants.OperatorDbName, constants.OperatorGtidsTableName))
+            name varchar(64) PRIMARY KEY,
+            value varchar(512) NOT NULL
+		)`, constants.OperatorDbName, constants.OperatorStatusTableName))
```

Comment:
Set this table engine as CSV so that it can be easily read by other applications as well.

Reply:
> To determine whether the new row already exists in the table, MySQL uses `PRIMARY KEY` or  `UNIQUE KEY index`. If the table does not have one of these indexes, the `REPLACE` statement is equivalent to the `INSERT` statement.

CSV table engine does not support `indexing` nor `PRIMARY KEY`